.github/workflows/cypress.yml

```diff
@@ -99,9 +99,6 @@ jobs:
     - name: Copy .htaccess
       run: npm run copy-htaccess
 
-    - name: Test
```

Comment:
Looks like the wrong step was removed. This test step should stay and I think the step above should be removed

Reply:
Fixed in 9b7c613c9e14fea43efad76196010d5e7c4054d5, I force pushed the incorrect commit to oblivion to keep the commit history clean.